include/class_loader/class_loader.hpp

```diff
@@ -59,25 +59,22 @@
 namespace class_loader
 {
 
-/// Returns the default library prefix for the native os
-CLASS_LOADER_PUBLIC
-std::string systemLibraryPrefix();
-
-/// Returns runtime library extension for native os
-CLASS_LOADER_PUBLIC
-std::string systemLibrarySuffix();
-
 /// Returns a platform specific version of a basic library name
 /**
  * On *nix platforms the library name is prefixed with `lib`.
  * On all platforms the output of class_loader::systemLibrarySuffix() is appended.
```

Comment:
`systemLibrarySuffix()` no longer exists right?

Reply:
Yes, I forgot to remove it